src/js/core/widget/core/PanelChanger.js

```diff
@@ -83,6 +84,28 @@
 			PanelChanger.classes = classes;
 			PanelChanger.prototype = prototype;
 
+			function writeToCookie(storageName, value) {
```

Comment:
Those functions looks like good candidates for utils since they could be used by different modules. What do you think?

Reply:
Ok. As above, I move these functions to 'tau.util.cookie'